addons/account/static/src/js/tax_group.js

```diff
@@ -39,19 +39,28 @@ odoo.define('account.tax_group', function (require) {
             } else {
                 creditAmount = -line_id.data.credit - deltaAmount; 
             }
-            // If debitAmount is negative, the value is added to the credit.
-            if (debitAmount < 0) {
-                creditAmount = Math.abs(debitAmount);
-                debitAmount = 0;
-            } else if (creditAmount > 0) {
-                debitAmount = Math.abs(creditAmount);
-                creditAmount = 0;
+            var amount_currency = 0;
```

Comment:
These conditions should be dependent of the invoice's type but... you don't have this value I guess...